src/utils/BrambleExtensionLoader.js

```diff
@@ -60,7 +61,8 @@ define(function (require, exports, module) {
     var extraExtensions = [
         "brackets-cdn-suggestions",    // https://github.com/szdc/brackets-cdn-suggestions
         "ImageUrlCodeHints",
-        "HTMLHinter"
+        "HTMLHinter",
+        "Autosave"
```

Comment:
Good catch, I forgot to remove it from here.  My thinking was to put it in the default list, and yes, you'd be able to disable with `?disableExtensions=Autosave`. I'll remove from extra.